### ISSUE TITLE
my ps4 controller the stick moves on its own in some games

### ISSUE THREAD

**Issue by EmilyNoceda** [FIX94/Nintendont#1222]
my ps4 controller the stick moves on its own in some games like it dont move in super mario sunshine, mario kart or paper mario but in zelda collections, sonic collection and sonic adventure move. ( and a unoficial port of sm64 too) idk if is my controller  any help?
if there`s a .ini of ps4 controller that`s solves this ploblem pleaseee :c

**Comment by carnage702** [comment]
most likely your controller has drift issue on the joysticks, and games that are coded more tight with controls will register the drift.you would have to increaze the deadzone but i dont know if you can even do that on an ini file but only on nintendont code and recompilling it.

**Comment by EmilyNoceda** [comment]
sooo... i need to change something of the .ini?

**Comment by carnage702** [comment]
> sooo... i need to change something of the .ini?

i dont think you can increase deadzones on the ini files sadly, you would need to alter nintendont itself and recompile it.
